tools/representations.py

```diff
@@ -139,6 +142,20 @@ def get_latest_datasets(self):
             if source.has_latest_dataset()
         }
 
+    def get_sources_by_feature(self, feature):
+        return {
+            source_id: source.as_json
```

Comment:
We are missing the parentheses here.
```suggestion
            source_id: source.as_json()
```